packages/website/src/routes/Landing/index.tsx

```diff
@@ -188,7 +212,7 @@ const styles = (theme: Theme) =>
         height: 40,
       },
     },
-    registerButton: {
+    whiteColorButton: {
       color: "#ffffff",
       border: "2px solid #ffffff",
```

Comment:
```suggestion
      color: "white",
      border: "2px solid white",
```
Just a small touch while we're here, completely fine to leave out :+1: